app/models/harvest.rb

```diff
@@ -9,7 +9,6 @@ class Harvest < ActiveRecord::Base
   belongs_to :plant_part
   belongs_to :planting
 
-  default_scope { order('created_at DESC') }
```

Comment:
I've added this default scope back in, and added a `reorder()` to plant_parts.harvests. this does what you'd expect unscoped to do.